CMSgov/easi-app
# [EASI-3202] Fix trb recipient emails

# EASI-3202

## Changes and Description

- Fixes the mutation for sending a TRB Advice letter to honor the input variable(s) for who to send notification emails to

## How to test this change

- Create a TRB Request
- Add some attendees (Gary Gordon, Adeline Aarons, etc.)
- Schedule a TRB Consult session in the past (so you can immediately start an advice letter)
- When sending the advice letter, ensure you can check/uncheck specific individuals for receiving notification, and use mailcatcher at http://localhost:1080 to validate who it's sending to.

## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
